src/services/recently-viewed.service.ts

```diff
@@ -0,0 +1,40 @@
+export class RecentlyViewedService {
```

Comment:
Can we name this a bit more specifically? There may come a time where there are other recently viewed item types.